tests/integration/targets/sysrc/tasks/cleanup-test-entries.yml

```diff
@@ -0,0 +1,23 @@
+---
+# #2319 help prevent failing tests from existing entries
+- name: Remove test entries from /etc/rc.conf
+  lineinfile:
+    path: /etc/rc.conf
+    state: absent
+    regexp: "^{{ pattern }}"
+  loop:
+    - "mysql_pidfile="
+    - "sysrc_absent="
+    - "sysrc_delim="
+    - "sysrc_delim_create="
+    - "sysrc_value_absent_delete="
+    # ezjail adds cloned_interfaces=lo1 so we only remove our entry
+    - 'cloned_interfaces="lo0 gif0"'
+  loop_control:
+    loop_var: pattern
+
+- name: Remove test entry from /boot/loader.conf
+  lineinfile:
```

Comment:
Also for `/boot/loader.conf`.

Reply:
I added the looping vars to the `/boot/loader.conf` version, but I'm not sure we need this file any more since I am using your other idea to copy/restore these two files.

Reply:
I guess you don't need, please remove this file.